### DISCUSSION TITLE
如何修改创建时的默认端口？

### DISCUSSION THREAD

**Discussion by feixu0530** [IceWhaleTech/CasaOS#721] [Q&A]
我在第一次安装时，安装命令默认安排了81端口。
但是我在今天再次安装时，安装命令默认安装在了80端口。80是比较常用的端口，我系统将它空出给其他程序使用。所以我有什么方法能够修改这个默认的端口？

**Comment by wangking6666** [comment]
I also want to know, but why hasn't anyone answered? Has the problem been resolved?

**Comment by junyiz** [comment]
试试修改 /etc/casaos/gateway.ini 里的 port
